plugin/aws/ec2InstanceMetadataJSON.go

```diff
@@ -23,7 +23,7 @@ func newEC2InstanceMetadataJSON(inst *ec2Instance) *ec2InstanceMetadataJSON {
 }
 
 func (im *ec2InstanceMetadataJSON) Open(ctx context.Context) (plugin.SizedReader, error) {
```

Comment:
Could we turn off caching for Open here? How much overhead would prettifying the metadata impose on performance?

Reply:
I think we could, it should be very minimal overhead.